net.sf.eclipsecs.doc/pom.xml

```diff
@@ -9,11 +9,31 @@
     <artifactId>net.sf.eclipsecs.doc</artifactId>
     <packaging>eclipse-plugin</packaging>
     <name>eclipse-cs Documentation Plugin</name>
-    <properties>
-        <timestamp>${maven.build.timestamp}</timestamp>
-    </properties>
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>calculate-website-timestamp</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <exec outputproperty="doctimestamp" executable="git">
+                                    <arg value="log"/>
+                                    <arg value="-1"/>
+                                    <arg value="--pretty=&quot;format:%cs&quot;"/>
```

Comment:
It looks like `format` is being printed into the file on generation with quotes.

https://github.com/checkstyle/eclipse-cs/actions/runs/4645708368/jobs/8221697711?pr=497#step:6:26
-    <lastmod>2023-04-08</lastmod>
+    <lastmod>"format:2023-04-08"</lastmod>

Reply:
Thanks for finding that. I now see that I can do the same without the additional quotes, as long as there are no spaces in the format.